app/actions/UserActions.js

```diff
@@ -383,6 +383,19 @@ export function createUser(token: string, user: string): Thunk<*> {
     });
 }
 
+export function deleteUser(username: string): Thunk<any> {
+  return callAPI({
+    types: User.DELETE,
+    endpoint: `/users/${username}/delete_user`,
+    method: 'DELETE',
+    schema: userSchema,
+    meta: {
+      errorMessage: 'Sletting av bruker feilet',
+    },
+    body: {},
```

Comment:
Is there a reason for the empty body key?

Reply:
Thank you for checking out the code, but it is very WIP. #hacktoberfest

Reply:
Also yes, it is empty because when issuing a HTTP delete we do not want to send any data in the body.